README.md

```diff
@@ -1,14 +1,42 @@
-A Go interface to an Elgato StreamdeckXL (specifically the 32-button version)
+# Go Streamdeck
+
+A Go interface to an Elgato Streamdeck.
```

Comment:
Should probably revert to say "Elgato Streamdeck XL (specifically the 32-button version)" until other devices are supported.